src/ts/components/transaction/Send.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react'
+import BN from 'bn.js'
+import { signExtrinsic } from '../../services/keyring-vault-proxy'
+import ApiPromise from '@polkadot/api/promise'
+import { RouteComponentProps, withRouter } from 'react-router'
+import {
+  Button as StyledButton,
+  ContentContainer,
+  Section
+} from '../basic-components'
+import { IAppState } from '../../background/store/all'
+import { connect } from 'react-redux'
+import 'react-tippy/dist/tippy.css'
+import { Form } from 'semantic-ui-react'
+import t from '../../services/i18n'
+import formatBalance from '@polkadot/util/format/formatBalance'
+import { IExtrinsic } from '@polkadot/types/types'
+import { SignerOptions } from '../../background/types'
+import { Index } from '@polkadot/types'
+import { SubmittableResult } from '@polkadot/api'
+
+interface ISendProps extends StateProps, RouteComponentProps, DispatchProps {
+}
+
+interface ISendState {
+  amount: BN,
+  toAddress: string,
+  hasAvailable: boolean,
+  isSi: boolean,
+  siUnit: string
+}
+
+const TEN = new BN(10)
+
+const si = formatBalance.findSi('m')
```

Comment:
default to milli unit for now.

Reply:
I am trying to add dropdown UI component, I would like to hear your opinion.

Reply:
yeah. we can have a dropdown for units. the dropdown value will be passed to `formatBalance.findSi({unitType})`
```export const SI: Array<SiDef> = [
  { power: -24, value: 'y', text: 'yocto' },
  { power: -21, value: 'z', text: 'zepto' },
  { power: -18, value: 'a', text: 'atto' },
  { power: -15, value: 'f', text: 'femto' },
  { power: -12, value: 'p', text: 'pico' },
  { power: -9, value: 'n', text: 'nano' },
  { power: -6, value: 'µ', text: 'micro' },
  { power: -3, value: 'm', text: 'milli' },
  { power: 0, value: '-', text: 'Unit' }, // position 8
  { power: 3, value: 'k', text: 'Kilo' },
  { power: 6, value: 'M', text: 'Mega' },
  { power: 9, value: 'G', text: 'Giga' },
  { power: 12, value: 'T', text: 'Tera' },
  { power: 15, value: 'P', text: 'Peta' },
  { power: 18, value: 'E', text: 'Exa' },
  { power: 21, value: 'Z', text: 'Zeta' },
  { power: 24, value: 'Y', text: 'Yotta' }
];```
https://github.com/polkadot-js/common/blob/master/packages/util/src/format/si.ts